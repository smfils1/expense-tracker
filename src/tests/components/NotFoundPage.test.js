import React from 'react'
import {shallow} from 'enzyme'
import NotFoundPage from "../../components/NotFoundPage"

describe('<ExpenseDashboard/>',()=>{
    test('Should render not found page correctly',()=>{
        const wrapper = shallow(<NotFoundPage/>)
        expect(wrapper).toMatchSnapshot()
    })
})