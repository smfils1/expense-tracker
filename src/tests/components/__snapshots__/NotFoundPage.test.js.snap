// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExpenseDashboard/> Should render not found page correctly 1`] = `
<h1>
  404
</h1>
`;
